velox/docs/functions/spark/map.rst

```diff
@@ -0,0 +1,34 @@
+===========================
+Map Functions
+===========================
+
+.. function:: getmapvalue(map(K,V), key) -> V
+    :noindex:
+
+    Returns value for given ``key``, or ``NULL`` if the key is not contained in the map.
+
+.. function:: element_at(map(K,V), key) -> V
+    :noindex:
+
+    Returns value for given ``key``, or ``NULL`` if the key is not contained in the map.
+
+.. function:: map() -> map<unknown, unknown>
+
+    Returns an empty map. ::
+
+        SELECT map(); -- {}
+
+.. function:: map(array(K), array(V)) -> map(K,V)
+
+    Returns a map created using the given key/value arrays. ::
+
+        SELECT map(ARRAY[1,3], ARRAY[2,4]); -- {1 -> 2, 3 -> 4}
+
+    See also :func:`map_agg` for creating a map as an aggregation.
```

Comment:
Will this link go to Presto doc? That's probably not what we want.

Reply:
Thank you for your carefulness. You're right. I need to use ‘spark_func’ to fix this issue.